benches/ironcore_alloy_bench.rs

```diff
@@ -65,106 +65,106 @@ fn benches(c: &mut Criterion) {
         };
         sdk.vector().encrypt(vector, &metadata).await.unwrap();
     };
-    c.bench_function("Standalone - vector_encrypt d=384", |b| {
```

Comment:
Why are all these just commented out now?

Reply:
Didn't mean to add that to the commit